core/trino-main/src/main/java/io/trino/operator/output/PagePartitioner.java

```diff
@@ -424,6 +457,7 @@ public void close()
     {
         try {
             flushPositionsAppenders(true);
+            outputSizeReportedBeforeRelease = 0;
```

Comment:
what would happen if this is not zeroed?

Reply:
Presumably nothing, this is just precautionary to meet expected semantics for what `close()` should do.